torch/csrc/distributed/rpc/py_rref.cpp

```diff
@@ -8,6 +8,58 @@
 namespace torch {
 namespace distributed {
 namespace rpc {
+
+/////////////////////  Pickle/Unpickle Helplers ////////////////////////////
+
+namespace {
+
+namespace {
```

Comment:
Do we need this nested anonymous namespace here?

Reply:
No, I think, one anonymous namespace should have confined names into the same file.